File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JsonResponseClasses/IntArray1DResponse.vb
Namespace ASCOM.Utilities.Support
    Public Class IntArray1DResponse
        Inherits RestResponseBase

        Private intArray1D As Integer()

        Public Sub New()
        End Sub

        Public Sub New(ByVal clientTransactionID As UInteger, ByVal transactionID As UInteger, ByVal value As Integer())
            ServerTransactionID = transactionID
            MyBase.ClientTransactionID = clientTransactionID
            intArray1D = value
        End Sub

        Public Property Value As Integer()
            Get
                Return intArray1D
            End Get
            Set(ByVal value As Integer())
                intArray1D = value
            End Set
        End Property
    End Class
End Namespace

File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JSON Response Classes/IntArray1DResponse.cs

namespace ASCOM.Utilities
{
    internal class IntArray1DResponse : RestResponseBase
    {

        private int[] intArray1D;

        public IntArray1DResponse()
        {
        }

        public IntArray1DResponse(uint clientTransactionID, uint transactionID, int[] value)
        {
            ServerTransactionID = transactionID;
            ClientTransactionID = clientTransactionID;
            intArray1D = value;
        }

        public int[] Value
        {
            get
            {
                return intArray1D;
            }
            set
            {
                intArray1D = value;
            }
        }
    }
}